lib/admin/end_drawer.dart

```diff
@@ -153,8 +163,15 @@ class _EndDrawerState extends State<EndDrawer> {
               trailing: Consumer<MaximoServerNotifier>(
                 builder: (context, maximo, child) {
                   return ElevatedButton(
-                    onPressed: () =>
-                        maximoAssetCaller(siteid, maximo.maximoServerSelected),
+                    onPressed: () {
+                      if(siteid != '') {
+                        maximoAssetCaller(siteid, maximo.maximoServerSelected); //TODO: pause user input during load?
+                        ScaffoldMessenger.of(context).showSnackBar(SnackBar(
+                            content: Text(
+                                'Attempting to Load Assets from : $siteid'),
+                          ));
+                      }
```

Comment:
I don't think that is necessary since users can load another site in the mean time or navigate to a different page